x-pack/plugins/apm/typings/Span.ts

```diff
@@ -22,9 +22,14 @@ export interface HttpContext {
   url?: string;
 }
 
+export interface TagsContext {
```

Comment:
Unused export

Reply:
would be nice if tslint supported this

Reply:
I actually looked at this recently :D
https://www.npmjs.com/package/ts-unused-exports